scaling_performance/optimizing_storage.adoc

```diff
@@ -11,19 +11,167 @@
+| Configurable
+| Configurable
+| Configurable
+| Configurable
+| Recommended
+
+| Object
+| Yes
+| Yes
+| Recommended
+| Recommended
+| Not configurable
+| Not configurable
+| Not configurable footnoteref:[disclaimer5, Object storage is not consumed via {product-title}'s PVs/PVCs. Apps must integrate with the object storage REST API.]
+|===
+
+[NOTE]
+====
+A scaled registry is an {product-title} registry where three or more pod replicas are running.
+====
+
+[[application-storage-recommendations]]
+==== Specific Application and Storage Recommendations
+
+[[registry]]
+===== Registry
+
+In a non-scaled/HA {product-title} registry cluster deployment, the preferred storage back-end type is object storage followed by block storage. The
+storage back-end type does not need to support ReadWriteMany (RWX) access mode. The storage back-end type must ensure read-after-write consistency. All
+NAS back-end types (excluding CNS/CRS GlusterFS as it uses an object storage interface) are expressly not recommended for {product-title} Registry cluster
+deployment with production workloads. While hostPath volumes are configurable for a non-scaled/HA {product-title} Registry cluster deployment, they are not recommended.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back {product-title} registry at production workloads.
+====
+
+[[scaled-registry]]
+===== Scaled Registry
+
+In a scaled/HA {product-title} registry cluster deployment, the preferred storage back-end type is object storage. The storage back-end type
+must support RWX access mode and must ensure read-after-write consistency. File storage and block storage are expressly not recommended
+for a scaled/HA {product-title} registry cluster deployment with production workloads. All NAS storage back-end types
+(excluding CNS/CRS GlusterFS as it uses an object storage interface) are expressly not recommended for {product-title} Registry cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back {production-title} scaled/HA registry at production workloads.
+====
+
+[[metrics]]
+===== Metrics
+
+In {product-title} hosted metrics cluster deployment, the preferred storage back-end type is block storage. All NAS storage back-end type (excluding
+CNS/CRS GlusterFS as it uses a block storage interface via iSCSI) are expressly not recommended for hosted metrics cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back hosted metrics at production workloads.
+====
+
+[[logging]]
+===== Logging
+
+In {product-title} hosted logging cluster deployment, the preferred storage back-end type is block storage. All NAS storage back-end type (excluding CNS/CRS GlusterFS as it
+uses a block storage interface via iSCSI) are expressly not recommended for hosted logging cluster deployment with production workloads.
+
+[WARNING]
+====
+Corruption may occur when using NFS to back Hosted Logging at production workloads.
```

Comment:
Should 'hosted logging' be capitalized? It's not in the previous section. Maybe it should be treated as a 'literal' in both places?